classes/tdx-signed-harden.inc

```diff
@@ -48,6 +48,7 @@ TDX_SECBOOT_REQUIRED_BOOTARGS_COMMON = "quiet logo.nologo vt.global_cursor_defau
 TDX_SECBOOT_REQUIRED_BOOTARGS_COMMON:append:cfs-support = " systemd.gpt_auto=0"
 
 TDX_SECBOOT_REQUIRED_BOOTARGS:imx-generic-bsp = "root=LABEL=otaroot rootfstype=ext4 ${TDX_SECBOOT_REQUIRED_BOOTARGS_COMMON}"
+TDX_SECBOOT_REQUIRED_BOOTARGS:apalis-imx8 = "pci=nomsi root=LABEL=otaroot rootfstype=ext4 ${TDX_SECBOOT_REQUIRED_BOOTARGS_COMMON}"
```

Comment:
I guess this was needed back in Kirkstone correct?

Reply:
Yes, I believe so. In fact the two commits:

- tdx-signed-harden.inc: update expected kargs for the apalis-imx8
- u-boot: update logic to detect bad usage of hardening

are likely needed on kirkstone. I intend to do it later after confirmation.